src/main/java/com/github/tomakehurst/wiremock/http/RequestMethod.java

```diff
@@ -75,7 +75,9 @@ public boolean equals(Object o) {
     if (o == null || getClass() != o.getClass()) return false;
 
     RequestMethod that = (RequestMethod) o;
-
+    if (name.equals("GET_OR_HEAD") && (that.name.equals("GET") || that.name.equals("HEAD"))) {
```

Comment:
I'm not sure doing the equals behaviour this way is a good idea. If we ever needed to put this e.g. as a map key I suspect this would prevent that from working properly.

Reply:
@tomakehurst Thanks for the feedback. Requested changes have been made.